hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/dao/NodeInfo.java

```diff
@@ -61,6 +61,7 @@
   private long usedMemoryOpportGB;
   private long usedVirtualCoresOpport;
   private int numQueuedContainers;
+  private int numPausedContainers;
```

Comment:
This all passes the information around but where are these number of paused containers actually modified?